lab-jonathan/index.js

```diff
@@ -0,0 +1,26 @@
+let express = require('express');
+let morgan = require('morgan');
+let jsonParser = require('body-parser').json();
+let PORT = process.env.PORT || 9000;
+
+// let createError = require('http-errors');
```

Comment:
You're on the right track with the error handling. You can put the function you have defined below into it's own module, require that in here, and then pass it to `app.use(errorHandler)` for the middleware functionality to take control of errors.